templates/projects/presenters/ner.html

```diff
@@ -0,0 +1,6 @@
+{% from "projects/_helpers.html" import render_project_card_option %}
+{{ render_project_card_option(project, upload_method, title=_('Named Entity Recognition'),
+   explanation=_('NER template'),
```

Comment:
I'm wondering if it could be helpful to use a longer descriptive text in the `explanation` field? This would be the text that displays along the bottom portion of the card, in which case, a longer text could be useful.

Just a suggestion! :)

![ner-card](https://user-images.githubusercontent.com/529049/58507923-db717480-8160-11e9-9533-9f6f31cf5559.png)

Reply:
changed